yaksh/forms.py

```diff
@@ -57,6 +58,7 @@ class UserRegisterForm(forms.Form):
                 (max_length=64, help_text='Department you work/study at')
     position = forms.CharField\
         (max_length=64, help_text='Student/Faculty/Researcher/Industry/etc.')
+    timezone = forms.ChoiceField(choices=[(tz, tz) for tz in pytz.common_timezones])
```

Comment:
Please set the default as UTC here, in case someone does not fill the Profile form or the Timezone field